next-cloudinary/package.json

```diff
@@ -2,13 +2,30 @@
   "name": "next-cloudinary",
   "version": "6.13.0",
   "license": "MIT",
-  "main": "./dist/index.js",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
-  "source": "src/index.ts",
+  "main": "dist/next-cloudinary.cjs.js",
+  "module": "dist/next-cloudinary.esm.js",
+  "exports": {
+    ".": {
+      "types": {
+        "import": "./dist/next-cloudinary.cjs.mjs",
+        "default": "./dist/next-cloudinary.cjs.js"
+      },
+      "module": "./dist/next-cloudinary.esm.js",
+      "import": "./dist/next-cloudinary.cjs.mjs",
+      "default": "./dist/next-cloudinary.cjs.js"
+    },
+    "./package.json": "./package.json",
+    "./cld-video-player.css": "./dist/cld-video-player.css",
+    "./dist/cld-video-player.css": "./dist/cld-video-player.css",
```

Comment:
do we need both of these entries for the CSS file and images?

Reply:
not really, I figured out that the shorter one might be preferred but I also had to add `dist/` to maintain the backwards compatibility. It's fine to drop the shorter one